Source/JavaScriptCore/runtime/JSTypedArrayViewConstructor.cpp

```diff
@@ -49,6 +49,7 @@ void JSTypedArrayViewConstructor::finishCreation(VM& vm, JSGlobalObject* globalO
 
     JSC_BUILTIN_FUNCTION_WITHOUT_TRANSITION(vm.propertyNames->of, typedArrayConstructorOfCodeGenerator, static_cast<unsigned>(PropertyAttribute::DontEnum));
     JSC_BUILTIN_FUNCTION_WITHOUT_TRANSITION(vm.propertyNames->from, typedArrayConstructorFromCodeGenerator, static_cast<unsigned>(PropertyAttribute::DontEnum));
+    globalObject->installTypedArrayConstructorSpeciesWatchpoint(this);
```

Comment:
don't we normally put these calls in JSGlobalObject itself?

Reply:
We are managing these watchpoints in JSGlobalObject (1) to suppress size increase of these constructors and (2) these constructors are only created per JSGlobalObject. And we already have this kind of methods for different types in JSGlobalObject. So I think JSGlobalObject is the right place.

Reply:
I'm asking if we should put this call inside JSGlobalObject instead of JSTypedArrayViewConstructor (just from a stylistic perspective)

Reply:
I think here is fine since (1) Set / Map prototypes are already doing the same thing. And (2) for Set and Map, it is hard to call it nicely since they are batch-initialized via `FOR_EACH_LAZY_BUILTIN_TYPE` macro. So, for consistency, I think putting here is better.

Reply:
Sounds good.